survey_link_base/models/survey_user_input.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Binhex - Adasat Torres de León
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SurveyUserInput(models.Model):
+    _inherit = "survey.user_input"
+
+    origin_model = fields.Char()
+    origin_id = fields.Integer()
```

Comment:
Maybe also add a reference field and add it to the user_input views to be able to go to the object that created the survey?